cmake/msvc/config.h

```diff
@@ -32,8 +32,9 @@ static inline float rintf(float x){return (x > 0.0f)? floorf(x + 0.5f) : ceilf(x
 ////////////////////////////////////////////////////////////////////////
 // math constants
 ////////////////////////////////////////////////////////////////////////
-#define INFINITY HUGE_VAL
-
+#if (!_MSC_VER >= 1900)
+# define INFINITY   HUGE_VAL
```

Comment:
Yes, if you can confirm said definition is present.  I believe that VS 2013 is the first math.h header that contains it, but could not find anything to absolutely confirm it and don't have it installed myself, so that's the only reason I only included 1900+